helper-cli/src/main/kotlin/commands/packageconfig/FormatCommand.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands.packageconfig
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.arguments.convert
+import com.github.ajalt.clikt.parameters.types.file
+
+import org.ossreviewtoolkit.helper.common.writeAsYaml
+import org.ossreviewtoolkit.model.config.PackageConfiguration
+import org.ossreviewtoolkit.model.readValue
+import org.ossreviewtoolkit.utils.expandTilde
+
+internal class FormatCommand : CliktCommand(
+    help = "Applies the formatting used by all ort-helper commands and strips all YAML comments. The output is " +
+            "written to the given package configuration file."
+) {
+    private val packageConfigurationFile by argument(
+        "package-configuration-file",
```

Comment:
The same parameter name is used in ORT for a file that contains a list of package configurations, I believe that could lead to some confusion.

Reply:
Well ok. In my view it became clearer that a single file for all package configurations is not a good idea.
The term 'package-configuration-file' is already used in 'helper-cli' in multiple places and this use here is consitent.
I prefer to keep it as-is now if that's fine with you?

I'd rather spend the time to remove the single file feature.

Reply:
Sounds good to me.